install/installation.md

```diff
@@ -26,53 +34,90 @@ cd /var/www/
-the sample-platform. After filling in these the platform should be ready for
-use.
+The `install.sh` will begin downloading and updating all the necessary
+dependencies. Once done, it'll ask to enter some details in order to set up
+the sample-platform. Fill in the the details.
+
+Install the requirements
+```
+sudo pip install -r requirements.txt
+sudo pip3 install -r requirements.txt
+```
+There might be an error something like libvirt not found. Ignore that
+```
+sudo apt-get install libvirt-dev
+```
+
+Running the program
+```
+sudo python3 run.py
+```
+
+If you face some problems try doing these -
+```
+In the config.py file change SERVER_NAME = '0.0.0.0:2000'
+In the run.py file change port = app.config.get('SERVER_PORT',2000)
+```
+
+```
+sudo mysql -u root -p
+use sample_platform;
+select * from general_data;
+```
+If you get an empty set: <br>
+  fill in some random data <br>
+  For example, <br>
+```
+INSERT INTO general_data (`key`, value) VALUES ('last_commit', '71dffd6eb30c1f4b5cf800307de845072ce33262');
+```
+close mysql <br>
+Try running the program again
 
 Please read the below troubleshooting notes in case of any error or doubt.
 
 ### Windows
 
 * Install cygwin (http://cygwin.com/install.html). When cygwin asks which
- packages to install, select Python, MySql, virt-manager and openssh. If you 
- already have cygwin installed, you must run its setup file to install the new packages. Make sure the dropdown menu is set to Full, so you can all packages. To select one, click skip and it will change to the version number of the package. Use the end of [this](https://www.davidbaumgold.com/tutorials/set-up-python-windows/) tutorial for help on getting cygwin to recognize python. 
-* Start a terminal session once installation is complete. 
+ packages to install, select Python, MySql, virt-manager and openssh. If you
+ already have cygwin installed, you must run its setup file to install the new packages. Make sure the dropdown menu is set to Full, so you can all packages. To select one, click skip and it will change to the version number of the package. Use the end of [this](https://www.davidbaumgold.com/tutorials/set-up-python-windows/) tutorial for help on getting cygwin to recognize python.
+* Start a terminal session once installation is complete.
 * Install Nginx (see http://nginx.org/en/docs/windows.html)
 * Install XMing XServer and setup Putty for ssh connections.
-* Virt-manager can call ssh to make the connection to KVM Server and should be 
-able to run virsh and send commands to it. 
+* Virt-manager can call ssh to make the connection to KVM Server and should be
+able to run virsh and send commands to it.
 * Follow the steps from the Linux installation from within the Cygwin terminal
 to complete the platform's installation.
 
+## Screencast-Demo
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/hmNj3gbe5rM/0.jpg)](http://www.youtube.com/watch?v=hmNj3gbe5rM "CCExtractor - Sample Platform : Installation Tutorial")
```

Comment:
I really like the fact that you went to all the trouble, but it contains incorrect settings, and as I mentioned above, steps that should not be necessary. Can those things be fixed?